vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -79,20 +79,18 @@
       <SubSystem>Windows</SubSystem>
       <AdditionalOptions>-minpdbpathlen:256</AdditionalOptions>
       <!--
-        comsuppw.lib  - _com_util::ConvertStringToBSTR
-        Winhttp.lib   - WebSocket resource implementation
+        comsuppw.lib              - _com_util::ConvertStringToBSTR
+        WindowsApp_downlevel.lib  - Replaces the WindowsApp.lib link reference in Microsoft.Windows.CppWinRT.props
```

Comment:
We should be able to target 8.1 and above.

Reply:
Agreed. I'll ask whoever introduced that change.
May update this in a subsequent PR, due to the scope.
I just took the opportunity to consolidate the dependency list.

Reply:
Followed up with @aeulitz, who authored this change.
It should be safe to remove.

Reply:
Leaving for a subsequent change.
It is unverified whether those APIs are available in Windows 8.1.

Reply:
Docs at https://docs.microsoft.com/en-us/windows/win32/apiindex/windows-81-api-sets show `api-ms-win-core-winrt-error-l1-1-1.dll` and `api-ms-win-core-winrt-string-l1-1-0.dll` as available on Windows 8.1, but don't show `api-ms-win-core-winrt-error-l1-1-0.dll`.